examples/set-name-prefix/.expected/diff.patch

```diff
@@ -0,0 +1,51 @@
+diff --git a/function.yaml b/function.yaml
+deleted file mode 100644
+index 1f596af..0000000
+--- a/function.yaml
++++ /dev/null
+@@ -1,11 +0,0 @@
+-apiVersion: v1
+-kind: ConfigMap
+-metadata:
+-  name: my-config
+-  annotations:
+-    config.k8s.io/function: |
+-      container:
+-        image: gcr.io/kpt-fn/set-name-prefix:unstable
+-    config.kubernetes.io/local-config: "true"
+-data:
+-  prefix: dev-
+diff --git a/resources.yaml b/resources.yaml
+index a9cfd43..1173423 100644
+--- a/resources.yaml
++++ b/resources.yaml
+@@ -1,7 +1,7 @@
+ apiVersion: apps/v1
+ kind: Deployment
+ metadata:
+-  name: nginx-deployment
++  name: dev-nginx-deployment
+   labels:
+     app: nginx
+ spec:
+@@ -23,17 +23,17 @@ spec:
+ apiVersion: autoscaling/v2beta2
+ kind: HorizontalPodAutoscaler
+ metadata:
+-  name: nginx-hpa
++  name: dev-nginx-hpa
+ spec:
+   scaleTargetRef:
+-    name: nginx-deployment
++    name: dev-nginx-deployment
```

Comment:
Why this field get changed when the [nameprefix config only have `.metadata/name`](https://github.com/kubernetes-sigs/kustomize/blob/master/api/konfig/builtinpluginconsts/nameprefix.go#L9)?